### ISSUE TITLE
Testing multiple version of python.

### ISSUE THREAD

**Issue by zkanda** [facebook/PathPicker#81]
There's an excellent tox package that can test multiple version of python in one command.
The only requirement is for the user to have the version installed.

Say you have all of this:
```
python26 -V
python27 -V
python33 -V
python34 -V
```

You can run one command to automatically test all of it:
```
tox
```

I already have it running in my local branch, let me know if you think this is good and I'll create a PR.

**Comment by pcottle** [comment]
yeah that would be great -- what are the dependencies for `tox`? im assuming it can work with travis CI in #82 right? :)

**Comment by metavida** [comment]
Travis CI has built-in support for testing multiple python versions (and the test suite is currently passing for 2.6 through 3.4) so I don't think we would need to use tox within Travis CI.

**Comment by zkanda** [comment]
tox dependencies is just virtualenv and pylib.

```
Successfully installed py-1.4.27 tox-1.9.2 virtualenv-12.1.1
```

**Comment by pcottle** [comment]
Fixed by #86
